### PR TITLE
Configured points system to a class.

### PR DESCRIPTION
**Pull request by hwangbro**
Instead of having your bot make calls into individual points.py functions, I rewrote the module as a class instead. This class will live inside your bot. While this change might not seem useful at first, I think having the code written this way is a little cleaner and more modular, with better views of responsibility for each function/class.

The new Points class also keeps track of the points_dict internally, and uses less calls to load_file. To make sure that the json is at least somewhat updated, the class writes the current status of the points_dict into the json every time the points are updated (every 15 seconds), as well as right before the bot dies with KeyboardInterrupt.

### CODE REVIEW

File: points.py [hunk 1]
```diff
@@ -6,89 +6,55 @@
...
+	def remove_user(self, user: str):
+		self.points_dict.pop(user, None)
+
+	def get_points(self, user: str):
+		return self.points_dict[user]
+
+	def get_bot(self) -> list:
+		return sorted(self.points_dict.items(), key=lambda kv: kv[1])
+
+	def update_points(self, usernames = util.get_viewers(): [str]):
```
Comment (rexosorous) [inline points.py]: invalid syntax for `util.get_viewers():`
it has a problem with the colon

File: points.py [hunk 2]
```diff
@@ -6,89 +6,55 @@
...
+		# op takes in "+", "-", ...
+		ops = {"+": operator.iadd,
+	           "-": operator.isub,
+	           "*": operator.imul,
+	           "/": operator.ifloordiv}
+	    update_set = {user}
+	    if user in util.everyone:
+	    	viewers = util.get_viewers()
+	    	blacklist = util.load_blacklist()
+	    	update_list.update({v for v in viewers if v not in blacklist})
```
Comment (rexosorous) [inline points.py]: `update_list` not defined

File: points.py (hunk 1 above)
Comment (hwangbro) [reply points.py]: Fixed in 5492eb6

File: points.py (hunk 2 above)
Comment (hwangbro) [reply points.py]: Fixed in c46b4c4

**Pull request merged**